ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Steps for extracting and applying the kube_secrets ISV parameter
+# Expects the standard Prow environment variables to be set
+
+REHEARSAL_INSTALL_NAMESPACE="!create"
+# Example of valid pyxis URL: https://pyxis.api.redhat.com/v1/projects/certification/pid/ospid-55159875-7eb1-41b0-bd91-0638b73536f1
```

Comment:
nit: remove the example to the partner certification URL

Reply:
Removed.